packages/PHPUnit/src/Rector/ExceptionAnnotationRector.php

```diff
@@ -73,30 +73,23 @@ public function test()
         );
     }
 
-    public function isCandidate(Node $node): bool
+    public function getNodeTypes(): array
     {
-        if (! $node instanceof ClassMethod) {
-            return false;
-        }
-
-        if (! $this->isInTestClass($node)) {
-            return false;
-        }
-
-        foreach ($this->annotationToMethod as $annotation => $method) {
-            if ($this->docBlockAnalyzer->hasTag($node, $annotation)) {
-                return true;
-            }
-        }
-
-        return false;
+        return [ClassMethod::class];
     }
 
     /**
      * @param ClassMethod $classMethodNode
      */
     public function refactor(Node $classMethodNode): ?Node
     {
+        if (! $classMethodNode instanceof ClassMethod) {
```

Comment:
I don't think we need it. If the refactor method was called, it means we have an instance of `ClassMethod`, like returned in `getNodeTypes`. This should be fixed in all Rectors

Reply:
Yes yes, working on it!